### PR TITLE
fix: .reuse/dep5

### PR DESCRIPTION
**Pull request by v0lkc**
## Purpose
There's a typo in the debian license file.

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type

What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

* Test the code

```
```

## What to Check

Verify that the following are valid

* ...

### CODE REVIEW

File: .reuse/dep5 [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: terraform-provider-btp
 Upstream-Contact: CloudOrchestrator@sap.com
```
Comment (v0lkc) [inline .reuse/dep5]: @JohannesRu @lechnerc77 shall the cloudorchestrator inbox remain here?

Comment (lechnerc77) [reply .reuse/dep5]: I don't think so. We need an external facing DL for that

Comment (lechnerc77) [reply .reuse/dep5]: Request for email is created

Comment (lechnerc77) [reply .reuse/dep5]: @v0lkc : You can set the email to "TerraformProviderBTP@sap.com". Not yet fully active according to ticket, but released and fully up in 24h

**Pull request merged**